nekoyume/Assets/_Scripts/Game/Stage.cs

```diff
@@ -548,6 +548,12 @@ private IEnumerator CoStageEnd(BattleLog log)
                             }
                         }
                     }
+                    else if(_battleResultModel.WorldID <= 10000)
```

Comment:
```suggestion
                    else if (_battleResultModel.WorldID <= 10000)
```

Reply:
In mimis brunnr stage, can not use repeat?

Reply:
> In mimis brunnr stage, can not use repeat?

As the document, all features of this pr will be applied only at the HAS stage.